tests/testthat/test.similarity.R

```diff
@@ -54,7 +46,7 @@ test_that("Recommendations work with group", {
   expect_equal(as.character(res), c("a", "b"), "We got products from diffrent groups")
 
   res <- recommendSimilar(m, viewed.skus, values, exclude.same = F)
-  expect_equal(as.character(res), c("a", "d"), "We got products from the same group")
+  expect_equal(as.character(res), c("a", "b"), "We got products from the same group")
```

Comment:
I have made the similarity matrix in the test symmetric, as it supposed to be. otherwise it is confusing.